### ISSUE TITLE
On close/dismiss event

### ISSUE THREAD

**Issue by maciekgrzybek** [timolins/react-hot-toast#174]
I'm in a situation where I have to react when the toast is being removed after some timeout. I can't really see the option to do that. 
Happy to create a PR for that. Let me know if it's needed :)

**Comment by jaguardo** [comment]
Yes, this would be a very good enhancement... is there currently a workaround to make it happen?

**Comment by plfmoura-redraw** [comment]
Have these toast closing events already been added?

I need to do something after the toast closes, I wanted to get this callback event to do something...
